.github/workflows/sign-container.yml

```diff
@@ -0,0 +1,80 @@
+name: sign-container
+
+on:
+  workflow_call:
+    inputs:
+      gpg_key_id:
+        description: ID for the key to be used for signing
+        required: false
+        type: string
+        default: security@ni.com
+      image_tag:
+        description: Tag of image to be signed
+        required: true
+        type: string
+      registry:
+        description: The container registry to retrieve image
+        required: false
+        default: niartifacts.jfrog.io
+        type: string
+      registry_username:
+        description: User to connect to artifactory
+        required: true
+        type: string
+      signature_store_bucket:
+        description: AWS S3 bucket to hold signatures
+        required: true
+        type: string
+      signature_store_region:
+        description: AWS region for the signature store
+        required: false
+        type: string
+        default: us-east-1
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        description: Access key to use for s3 signature bucket
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        description: Secret access key to use for s3 signature bucket
+        required: true
+      GPG_PRIVATE_KEY:
+        description: Base 64 encoded gpg private key to use for signing
+        required: true
+      REGISTRY_PASSWORD:
+        description: Password or token to use to pull from container registry
+        required: true
+
+jobs:
+  sign-container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Log into registry
+        uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ inputs.registry_username }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+          registry: ${{ inputs.registry }}
+      - name: Create signature staging directory
+        run: mkdir ${{ runner.temp }}/sigstore
+      - name: Create gnupghome directory
+        run: mkdir ${{ runner.temp }}/gpg && chmod 700 ${{ runner.temp }}/gpg
+      - name: Import private key
+        run: echo "${{ secrets.GPG_PRIVATE_KEY }}" | base64 --decode | gpg --import
+        env:
+          GNUPGHOME: ${{ runner.temp }}/gpg
+      - name: List keys
+        run: gpg --list-secret-keys
```

Comment:
this is only for debugging so can be deleted?

Reply:
You are correct.  I'll remove that.